src/store/emptyImageBuilderApi.ts

```diff
@@ -5,6 +5,21 @@ import { IMAGE_BUILDER_API } from '../constants';
 // initialize an empty api service that we'll inject endpoints into later as needed
 export const emptyImageBuilderApi = createApi({
   reducerPath: 'imageBuilderApi',
-  baseQuery: fetchBaseQuery({ baseUrl: IMAGE_BUILDER_API }),
+  baseQuery: fetchBaseQuery({
+    baseUrl: IMAGE_BUILDER_API,
+    paramsSerializer: (params) => {
+      const searchParams = new URLSearchParams();
+      for (const [key, value] of Object.entries(params)) {
+        if (Array.isArray(value)) {
+          for (const v of value) {
+            searchParams.append(key, v);
+          }
+        } else {
+          searchParams.append(key, value);
+        }
+      }
+      return searchParams.toString();
+    },
+  }),
```

Comment:
Let's unpack a bit what's changed here, so basically this change implies that now, every request sent to the image builder backend will be executing this chunk of code.

I don't think that's a correct approach to doing the thing that you want. Especially since it has already consequences for the cloneCompose endpoint. In the future there might be other endpoints having the same issue and that's going to be super hard to find out what's going on.

So we have to find another way of doing things for this PR.